index.css

```diff
@@ -1,3 +1,21 @@
-html {
-  
+html {}
+.board { margin:1em auto; border-collapse:collapse }
+.board td {
+   cursor:pointer;
+   width:100px; height:100px;
+   border:5px solid #ccc;
+   text-align:center;
+   font-family:sans-serif; font-size:16px
+}
+
+/*table td {
+  border: 5px solid black;
+}*/
+
+.purple-rain {
```

Comment:
When naming styling classes, try to be as objective as possible. What if you changed the background color of this element to a yellow hue? `.purple-rain` wouldn't be descriptive anymore. Name classes after content, purpose, or object.